### ISSUE TITLE
error cubejs is not a function when loading cubejs-client-core.umd.js >= V0.20.0

### ISSUE THREAD

**Issue by garagevibes** [cube-js/cube.js#1180]
**Describe the bug**
I have been importing the core cube js client into my dashboard app but following an update i notice the distribution of cubejs-client-core.umd.js at version 0.20.0 has an error. version 0.19.56 works fine. The error shows as "cubejs is not a function" this is at the point of initialising the api.

**To Reproduce**
1. create a basic web page and add a script tag to include the cubejs client like so: <script src="https://unpkg.com/@cubejs-client/core@0.20.0/dist/cubejs-client-core.umd.js"></script>
2. create a quick function like var cubejsApi = cubejs(cubeJsToken,{ apiUrl: cubeJsAppUrl });
3. your get the above error
4. change the script tag to: <script src="https://unpkg.com/@cubejs-client/core@0.19.56/dist/cubejs-client-core.umd.js"></script> and it will work however thne i cant access the new functions like timeDimention compaire.

**Version:**
all versions greater than and including v0.20.0

**Comment by garagevibes** [comment]
it seems the issue is related to this function:

function getAugmentedNamespace(n) {
		if (n.__esModule) return n;
		var a = Object.defineProperty({}, '__esModule', {value: true});
		Object.keys(n).forEach(function (k) {
			var d = Object.getOwnPropertyDescriptor(n, k);
			Object.defineProperty(a, k, d.get ? d : {
				enumerable: true,
				get: function () {
					return n[k];
				}
			});
		});
		return a;
	}

if I comment out line 18737 - var index$2 = /*@__PURE__*/getAugmentedNamespace(src);

and add back  - var index$2 = ( src && index$1 ) || src;

as it was previously everything seems to work ok again. I haven't yet found the cause of the issue with the getAugmentedNamespace function but as this file is bundled by rollup it seems the issue could be related to the process there rather than the function itself as I note other implementations of the client seem to work fine.

**Comment by vasilev-alex** [comment]
Hey @garagevibes! Thanks for reporting.

This behavior changed after we upgraded the `rollup` version to the latest. Before, you could use only the default export (and not the rest) from the `umd` build. Currently, all exports are available at `cubejs` namespace.

e.g.

```js
const cubejsApi = cubejs.default(cubeJsToken,{ apiUrl: cubeJsAppUrl })
```

Please let me know what you think about it.

**Comment by vasilev-alex** [comment]
ok, actually we've decided to keep it backward compatible.

So, this remains the same

```js
const cubejsApi = cubejs(cubeJsToken, { apiUrl: cubeJsAppUrl });
```

But on top, `cubejs` will contain all other exports as well. 

e.g.

```js
cubejs.ResultSet(...);
cubejs.HttpTransport(...)
// etc.
```

Closed by #1219

**Comment by dc-thanh** [comment]
This error is really frustrating. I'm using cubejs-client-core version 0.34.37 with Vite and Svelte Kit, and I can't build my website while the dev mode is working fine.
![CleanShot 2024-01-13 at 10 08 51](https://github.com/cube-js/cube/assets/61302262/5aab8436-6c2e-4f6d-8413-2c55c8e44d06)
